packages/xpetra/src/Vector/Xpetra_TpetraVector_decl.hpp

```diff
@@ -197,50 +190,55 @@ namespace Xpetra {
     ///          only valid as long as the vector does not run of scope!
     template<class TargetDeviceType>
     typename Kokkos::Impl::if_c<
-      Kokkos::Impl::is_same<
-        typename dual_view_type::t_dev_um::execution_space::memory_space,
-        typename TargetDeviceType::memory_space>::value,
-        typename dual_view_type::t_dev_um,
-        typename dual_view_type::t_host_um>::type
-    getLocalView () const;
+      Kokkos::Impl::is_same<typename dual_view_type::t_dev_um::execution_space::memory_space, typename TargetDeviceType::memory_space>::value,
```

Comment:
In future commits, prefer `std::is_same` to `Kokkos::Impl::is_same`, and prefer `std::conditional` to `Kokkos::Impl::if_c`.